components/common/hooks/useCanSelfHostedSignup.ts

```diff
@@ -2,17 +2,24 @@
 // See LICENSE.txt for license information.
 
 import {useState, useEffect} from 'react';
+import {useSelector} from 'react-redux';
 
 import {Client4} from 'mattermost-redux/client';
+import {getConfig} from 'mattermost-redux/selectors/entities/general';
 
 import useLoadStripe from './useLoadStripe';
 
 export default function useCanSelfHostedSignup() {
     const [canReachPortal, setCanReachPortal] = useState(false);
```

Comment:
nit: very unrelated to your change, and "if it ain't broken don't fix it"....  but one hook use `canReachPortal` and the other one use `canReachCWS`. and I prefer CWS over the generic "portal" word 😅

Reply:
I'm a fan of changing to CWS and will eventually. Won't in this PR because its being backported and there's a few versions of this hook still in progress.